Makefile

```diff
@@ -29,7 +29,7 @@ help:
 .PHONY: services
 services: args?=up -d
 services: python
-	@tox -qe docker-compose -- $(args)
+	@./bin/services/are-up || (tox -qe docker-compose -- $(args) && ./bin/services/wait-until-up)
```

Comment:
This could easily be split into `services` and `services-up`. If you do them together you can skip a second check for whether they are up which would happen if they were.